pages/glossary/semantic-role.md

```diff
@@ -11,4 +11,19 @@ A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic
 
 Valid semantic roles are defined by standards. For web content and applications these are defined within the various [WAI-ARIA specifications](https://act-rules.github.io/glossary/#wai-aria-specifications).
 
-The semantic role of an element is its [explicit semantic role](#explicit-role) if it has any, otherwise, the [implicit semantic role](#implicit-role) is used.
+The _semantic role_ of an element is determined by the first of these cases that applies:
+
+1. **Conflict** If the element has an [explicit role][] which is either `none` or `presentation`, but the element is [included in the accessibility tree][], or would be when it is not [hidden](https://www.w3.org/TR/wai-aria-1.2/#dfn-hidden), then its _semantic role_ is its **[implicit role][]**.
```

Comment:
I'm not entirely sold on the objectiveness of the WCAG definition of "hidden". The way I interpret it is that an element is hidden when it doesn't generate any layout boxes, or generates only 0-dimension boxes, meaning that it isn't rendered. Thoughts?

Reply:
@kasperisager would this be better?
```suggestion
1. **Conflict** If the element has an [explicit role][] which is either `none` or `presentation`, but the element is [included in the accessibility tree][], or would be when it is [visible](#visible), then its _semantic role_ is its **[implicit role][]**.
```

Reply:
I'm pretty sure that's not the definition we'll want to use though. Per https://w3c.github.io/aria/#tree_exclusion, things are removed from the accessibility tree when they have either `display: none` or `visibility: hidden`.

Reply:
Then 
```suggestion
1. **Conflict** If the element has an [explicit role][] which is either `none` or `presentation`, but the element is [included in the accessibility tree][], or would be when it is not [excluded from the accessibility tree](https://w3c.github.io/aria/#tree_exclusion), then its _semantic role_ is its **[implicit role][]**.
```